projects/common/src/key-codes/key-code.ts

```diff
@@ -0,0 +1,7 @@
+export const enum KeyCode {
+  Enter = 'Enter',
+  Comma = ','
+}
+
+export const isEnterOrCommaKeyEvent = (event: KeyboardEvent): boolean =>
```

Comment:
Should we make this configurable? Enabled `Enter to add new` or `Comma to add new`?

Reply:
will add a general method in a follow up.